docs/components/concepts/clusters.md

```diff
@@ -0,0 +1,44 @@
+
+|              | Development | Production - S | Production - M | Production - L |
+| ------------ | ----------- | -------------- | -------------- | -------------- |
+| Free         | \-          | \-             | \-             | \-             |
+| Free Trial   | \-          | X              | \-             | \-             |
+| Professional | X           | X              | \-             | \-             |
+| Enterprise   | X           | X              | X              | X              |
+
+## Production clusters
+
+Production clusters come in three sizes: small (S), medium (M), and large (L). To learn more about the size of cluster best suited for your use case, see our [Best Practices](/components/best-practices/best-practices-overview.md) for more information on [sizing your runtime environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
+
+When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DMN](/components/modeler/dmn/dmn.md) models on a production cluster, this might impact your monthly (Professional) or annual (Enterprise) total fee, meaning the more you execute your models, the higher your total fee may be.
+
+### Free Trial cluster
+
+A Free Trial cluster is a special kind of production cluster. Free Trial clusters are size small, and only available during your trial period.
+
+Once you sign up for a Free Trial, you are able to spin up one production cluster for the time of your trial.
+
+When your trial expires, you automatically move to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster spun up during your free trial is deleted, and you cannot spin up new clusters.
+
+## Development clusters
+
+Development clusters are recommended for development, testing, proof of concepts, and demos.
+
+Professional Plan users have the option to create **development clusters**, offering free execution for development. Deployment and execution of models (process instances, decision instances, and task users) is provided at no cost.
+
+Additionally, the following applies to **development clusters**:
+
+- **Cluster is not High-Available & Less Hardware**: Reduced hardware resources and availability compared to production cluster (for example, one Zeebe node only). No 99.5% SLA, but still sufficient for most development use cases.
+- **Shorter history of processes and decisions**: Data retention in Operate, Optimize, and Tasklist is reduced to one day. For example, pending or historical process instances are deleted after one day.
+- **Cluster auto-pause**: We will pause your cluster if it's not used for two hours. You can wake up your cluster by clicking a button in Console. This means not all functionality will work all the time. For example, BPMN timers and BPMN message catch events will not work while clusters are paused.
```

Comment:
"We will pause your cluster if it's not used for two hours." -- To clarify, I would maybe reword this as something like "Your cluster is paused if it goes unused for two hours" or something of the like.

"You can wake up your cluster by clicking a button in Console." -- Do you mean you can click any button in Console to get your cluster working again?

Reply:
> "You can wake up your cluster by clicking a button in Console." -- Do you mean you can click any button in Console to get your cluster working again?

This is question back to Jakob or Felix. I'll follow up in the working doc.

Reply:
This is a future item, we'll need to remove it from the PR.